### ISSUE TITLE
CheXNet: Radiologist-Level Pneumonia Detection on Chest X-Rays with Deep Learning

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#709]
https://arxiv.org/abs/1711.05225

>We develop an algorithm that can detect pneumonia from chest X-rays at a level exceeding practicing radiologists. Our algorithm, CheXNet, is a 121-layer convolutional neural network trained on ChestX-ray14, currently the largest publicly available chest X-ray dataset, containing over 100,000 frontal-view X-ray images with 14 diseases. Four practicing academic radiologists annotate a test set, on which we compare the performance of CheXNet to that of radiologists. We find that CheXNet exceeds average radiologist performance on pneumonia detection on both sensitivity and specificity. We extend CheXNet to detect all 14 diseases in ChestX-ray14 and achieve state of the art results on all 14 diseases.

Some [good discussion](https://twitter.com/AndrewLBeam/status/930857531337035778) on [Twitter](https://twitter.com/AndrewYNg/status/930938692310482944).

**Comment by souravsingh** [comment]
@agitter Are we looking into adding this for the revised manuscript. If so, I can work on adding this.

**Comment by yfpeng** [comment]
I want to point that this paper is not peer-reviewed. Some even think it might be over-claimed. I would suggest to put it aside for now.

**Comment by agitter** [comment]
@souravsingh I've continued to open issues for relevant manuscripts so that we can discuss them and see if someone wants to make the case that a paper should be added to our review.  We are also trying to minimize changes to the manuscript until we address all of the issues raised by the reviewers and will prioritize merging pull request that address those comments.

**Comment by agitter** [comment]
@yfpeng not being peer-reviewed doesn't bother me, but you are correct in implying that we should not take the claims of this or any other paper at face value.

**Comment by souravsingh** [comment]
@agitter Makes sense. I can help in fixing the issues with manuscript.

**Comment by cgreene** [comment]
Some additional stuff here: https://lukeoakdenrayner.wordpress.com/2017/11/18/quick-thoughts-on-chestxray14-performance-claims-and-clinical-tasks/

**Comment by brettbj** [comment]
see also: #722
